src/natcap/invest/cli.py

```diff
@@ -275,15 +272,13 @@ def main(user_args=None):
     listmodels_subparser.add_argument(
         '--json', action='store_true', help='Write output as a JSON object')
 
-    subparsers.add_parser(
-        'launch', help='Start the InVEST launcher window')
-
     run_subparser = subparsers.add_parser(
         'run', help='Run an InVEST model')
+    # Recognize '--headless' for backwards compatibility.
```

Comment:
What would you think about gradually deprecating this option so we don't have to keep it around forever? We could warn the user that it's deprecated now and will be removed in a future release

Reply:
@emlys good idea! Just pushed the change.

Reply:
I tested it manually with all the variations of the command. I didn't bother adding a test to check for the logging since this isn't a feature we plan to keep around for long.